devops/jobs/ImageBuilder.groovy

```diff
@@ -0,0 +1,107 @@
+/*
+    This job serves to build an application's Docker image and to push it to Docker Hub. This job only serves as a downstream job to the 
+    various <app>-watcher jobs and does not run on its own. This job receives the name of an application as a build parameter. 
+    This application's Docker image is then built and pushed.
+
+    Upon a push to master, the job is triggered via a webhook, at which point it runs the parsefiles.py script with the files changed between 
```

Comment:
This doc is about the configuration job above, right?
This job is either called by ConfigurationWatcher or AppWatcher, right?